unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs

```diff
@@ -222,6 +237,33 @@ private void HandleMessageReceived(ChatMessage message)
             // The messages from 'conversationUserId' are marked as read if his private chat window is currently open
             MarkUserChatMessagesAsRead();
         }
+
+        deactivatePreviewCancellationToken.Cancel();
+        deactivatePreviewCancellationToken = new CancellationTokenSource();
+        deactivatePreviewHUDControllerCancellationToken.Cancel();
+        deactivatePreviewHUDControllerCancellationToken = new CancellationTokenSource();
+
+        if (_currentState.Equals(ChatWindowVisualState.INPUT_MODE))
+        {
+            //Input window is present. Start counter to deactivate
+            if (chatHudController.IsInputSelected)
+                return;
+            WaitThenActivatePreview(deactivatePreviewCancellationToken.Token).Forget();
+        }
+        else
+        {
+            //If the chatHudController is visible, just restart the counter
+            if (_currentState.Equals(ChatWindowVisualState.PREVIEW_MODE))
+            {
+                //Start Fade Out Counter
+                WaitThenDeactivatePreviewChatHUDController(deactivatePreviewHUDControllerCancellationToken.Token).Forget();
+            }
+            else
```

Comment:
i would specify the case: `else if (currentState.Equals(ChatWindowVisualState.NONE_VISIBLE))` since its easier to follow